rerun_py/tests/api_sandbox/rerun_draft/catalog.py

```diff
@@ -58,22 +74,6 @@ def table_names(self) -> list[str]:
         """Returns a list of all table names in the catalog."""
         return self._inner.table_names()
 
-    def entries(self) -> datafusion.DataFrame:
-        """Returns a DataFrame containing all entries in the catalog."""
-        return self._inner.entries()
-
-    def datasets(self) -> datafusion.DataFrame:
-        """Returns a DataFrame containing all dataset entries in the catalog."""
-        return self._inner.datasets()
-
-    def tables(self) -> datafusion.DataFrame:
-        """Returns a DataFrame containing all table entries in the catalog."""
-        return self._inner.tables()
```

Comment:
Are we 100% sure these weren't useful? Would be nice to have confirmation from @ntjohnson1 .